src/stores/Create.ts

```diff
@@ -96,6 +111,8 @@ export interface CreateEntityStore {
     setConnectors: (val: CreateEntityStore['connectors']) => void;
     endpointSchema: { [key: string]: any };
     setEndpointSchema: (val: CreateEntityStore['endpointSchema']) => void;
+    isIdle: boolean;
+    isActive: boolean;
```

Comment:
Initially the active/idle was not symmetrical. Eventually it became so and now having two flags for this is not really needed but I left it that way.